README.md

```diff
@@ -55,67 +23,52 @@ After you're finished please remove all the comments and instructions!
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+# 📖 Hello Microverse <a name="about-project"></a>
 
-**[your_project__name]** is a...
+**Hello Microverse** is my first project in Microverse, done in Block 1 of Module 1.
 
 ## 🛠 Built With <a name="built-with"></a>
 
-### Tech Stack <a name="tech-stack"></a>
-
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
+### HTML-Used for Text & CSS-Used for styling<a name="tech-stack"></a>
 
 <details>
-  <summary>Server</summary>
+  <summary>Markup Language</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="https://html.com/">HTML</a></li>
   </ul>
 </details>
 
 <details>
-<summary>Database</summary>
+  <summary>Styling Language</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://web.dev/learn/css/">CSS</a></li>
   </ul>
 </details>
 
-<!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Displays Hello Microverse!**
+- **Backgroud Colour is Cyan**
+- **Text Size is h1**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
+- [Coming Soon]("")
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps: Clone Github Repo using SSH, HTTP, or via Github Desktop.
+
+
 
 ## 💻 Getting Started <a name="getting-started"></a>
```

Comment:
- **[OPTIONAL]** The getting started section is duplicated here, please stick to only one version so the documentation will look more professional 👍🏼